src/main/java/org/spongepowered/api/world/explosion/Explosion.java

```diff
@@ -92,30 +93,37 @@ static Builder builder() {
     boolean shouldDamageEntities();
 
     /**
-     * Gets the number of rays cast per length of each side of the explosion bounding box.
+     * Gets a value that indicates the relative strength of an explosion.
      *
-     * @return The number of rays cast per length of each side of the explosion bounding box
+     * @return The resolution of the explosion.
      */
     default int getResolution() {
         return 16;
     }
 
     /**
-     * <pre>
-     * A numerical representation of variation in blast strength per ray.
-     * The blast strength of a ray is calculated as {@code (1 * [variation]) * [radius]}
-     * 0.0 = No Variation.
-     * 1.0 = +/-0.3 Variation (Vanilla Behaviour).
-     * </pre>
+     * Gets an indication of the randomness of the form of the explosion.
      *
-     * @return A numerical representation of variation in blast strength per ray
+     * <p>This value indicates how close to being symmetrical the explosion is.
+     * A value of zero indicates a completely symmetrical blast (in all three
+     * dimensions). A larger value indicates a lower likelihood of symmetry.</p>
```

Comment:
Missing `A value of one indicates the platform default.` (copied from the method below)